common/middleware/src/line_rules.rs

```diff
@@ -0,0 +1,260 @@
+use crate::{Middleware, Status};
+use http::types::body::LineMetaMut;
+use regex::bytes::Regex as BytesRegex;
+use regex::Regex as TextRegex;
+use thiserror::Error;
+
+static REDACTED_TEXT: &str = "[REDACTED]";
+static REDACT_BYTES: &[u8] = &REDACTED_TEXT.as_bytes();
+
+pub struct LineRules {
+    exclusion_text: Vec<TextRegex>,
```

Comment:
Do we need the TextRegexs? Could we not call as_bytes() on String lines and treat everything as bytes?

Reply:
Good idea, I'll give it a try.